examples/traces/values-config.yaml

```
@@ -0,0 +1,29 @@
+global:
+  observe:
+    collectionEndpoint: ""
+
+observe:
+  token:
+    value: ""
+
+opentelemetry-collector:
+  # overrides pod resource requests/limits
+  resources:
+    limits:
+      cpu: 250m
+      memory: 512Mi
+    requests:
+      cpu: 250m
+      memory: 512Mi
+
+  # overrides collector configuration
+  config:
+    processors:
+      # https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiterprocessor/README.md
+      memory_limiter:
+        # to use percentagles instead of the _mib values
+        limit_percentage: 75
+        spike_limit_percentage: 20
+        # may need to explicitly unset default values specified
```

```suggestion
        # unset default values
```

Not _may_, _must_:

https://github.com/observeinc/helm-charts/blob/44ad42766b0157a5c13ce1466e1cbcd0af2ef2cb/charts/traces/values.yaml#L145-L147

How the application handles both % and absolute values in config would take further looking but it's at best ambiguous and preferably forbidden.